Lib/venv/__init__.py

```diff
@@ -4,18 +4,38 @@
 Copyright (C) 2011-2014 Vinay Sajip.
 Licensed to the PSF under a contributor agreement.
 """
+import configparser
 import logging
 import os
 import shutil
 import subprocess
 import sys
 import sysconfig
 import types
-
+from typing import NamedTuple, Optional
 
 CORE_VENV_DEPS = ('pip', 'setuptools')
 logger = logging.getLogger(__name__)
 
+if os.name == "nt":
+    _dir = os.path.expandvars(r'%APPDATA%\Python')
```

Comment:
Pro-tip: if your raw-string is not a regular expression and you write R'...' instead of r'...' then GitHub will highlight it better.

Reply:
TIL

Reply:
It seems in `vscode` too in case you didn't know :D